govcd/sample_govcd_test_config.yaml

```diff
@@ -77,6 +77,15 @@ vcd:
     # An external Network name
     externalNetwork: myexternalnet
     #
+    # A port group name for creating an external network
+    externalNetworkPortGroup: ForTestingPG
+    #
+    # A port group type for creating an external network. Can be DV_PORTGROUP or NETWORK
```

Comment:
We should specify that these fields are used for a new external network to be used in tests.

Reply:
Fixed